flaml/tune/tune.py

```diff
@@ -236,9 +236,21 @@ def compute_with_config(config):
             respectively. You can also provide a self-defined scheduler instance
             of the TrialScheduler class. When 'asha' or self-defined scheduler is
             used, you usually need to report intermediate results in the evaluation
-            function. Please find examples using different types of schedulers
+            function via 'tune.report()'. In addition, when 'use_ray' is not enabled,
+            you also need to stop the evaluation function according to the returned result
+            of 'tune.report()' as shown in the following example.
+            Please find more examples using different types of schedulers
             and how to set up the corresponding evaluation functions in
-            test/tune/test_scheduler.py. TODO: point to notebook examples.
+            test/tune/test_scheduler.py, and test/tune/example_scheduler.py.
+    ```python
+    def easy_objective(config):
+        width, height = config["width"], config["height"]
+        for step in range(config["steps"]):
+            intermediate_score = evaluation_fn(step, width, height)
+            trial_not_finished = tune.report(iterations=step, mean_loss=intermediate_score)
+            if trial_not_finished is False:
+                return

```

Comment:
I'm not sure if we should return None or return the intermediate_score. Could you check?